sotodlib/tod_ops/demodulation.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
+from sotodlib.tod_ops import filters
+from scipy.optimize import curve_fit
+import matplotlib.pyplot as plt
```

Comment:
matplotlib imports like this cause issues in modules because people use/connect to them in so many different ways. Can we move this one to inside the plotting if statements?

Reply:
makes sense, done